### PR TITLE
Rgb noise models

### PR DESCRIPTION
**Pull request by NicoGrande**
## Motivation and Context

This change builds upon the redwood depth noise model implemented by Erik Wijmans with the objective of implementing color noise models in Habitat. This change includes initial commits for four new RGB noise models, namely Gaussian noise, Speckle noise, Salt and Pepper noise and Poisson noise. The hope for this pull request is to continue the efforts to allow for embodied agents to train under realistic sensor noise to further understand their behavior in sub-optimal sensor conditions. 

## How Has This Been Tested

These modification have been tested to gauge the frame rate at which the noise models are applied to arbitrary NumPy arrays as well as tested to see the visual effect of the noise models in the simulation environment. These tests can be replicated using sensor_test.py and viewer.py respectively. 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by mathfac** [comment]
Hi @NicoGrande, looks like you need to properly rebase on master.

File: habitat_sim/sensors/noise_models/registration.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import abc
```
Comment (erikwijmans) [inline habitat_sim/sensors/noise_models/registration.py]: I believe this file can be removed

File: viewer.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import cv2
```
Comment (erikwijmans) [inline viewer.py]: Please remove this file from the PR

File: sensor_test.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import timeit
```
Comment (erikwijmans) [inline sensor_test.py]: Same with this file

**Comment by mathfac** [comment]
Added noise models RGB outputs with default parameters to make noise mode choice easier.

**Pull request merged**